lib/generators/phlex/install/install_generator.rb

```diff
@@ -4,8 +4,25 @@ module Phlex::Generators
 	class InstallGenerator < ::Rails::Generators::Base
 		source_root File.expand_path("templates", __dir__)
 
+    IGNORED_TAILWIND_CONFIGURATION_LOOKUP_DIRECTORIES = %w[
+      .git
+      .github
+      .ruby-lsp
+      bin
+      coverage
+      db
+      docs
+      log
+      node_modules
+      public
+      spec
+      storage
+      test
+      tmp
+      vendor
+    ].freeze
```

Comment:
I don’t like the idea of Phlex maintaining an ignore list that users can’t update. Instead, could we look in just `app/**` and `config/**`, ignoring everything else?

Reply:
Totally! See updated PR. 

> could we look in just app/** and config/** 

I also added "#{Rails.root}/tailwind.config.js" for good measure 😅